testsuite/features/step_definitions/common_steps.rb

```diff
@@ -647,7 +647,7 @@
   # Preparation of bootstrap script for different types of clients
   client = client_type == 'traditional' ? '--traditional' : ''
   # Use server if proxy is not defined as proxy is not mandatory
-  target = (target_type.include? 'server') || $proxy.nil? then $server else $proxy
+  target = if (target_type.include? 'server') || $proxy.nil? then $server else $proxy
```

Comment:
Opps, my bad! This is not Python, so we must close the `if` expression (i.e., we must add `end`):
`if (target_type.include? 'server') || $proxy.nil? then $server else $proxy end`